projects/go-lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tangoe/goponents",
-  "version": "0.5.5",
+  "version": "0.5.6",
```

Comment:
Should this be ticked up for every addition? Or should it be ticked up on each release?

Reply:
I am considering this a hotfix, so I am incrementing the version so I can republish it. We do need to figure out the best way of how this should work though. 🤷‍♂️

Reply:
It _seems_ like most open source projects will have a "Release PR" that contains this work. The [Angular Repo for example](https://github.com/sfeir-open-source/angular/pull/266/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R3).

Reply:
For the sake of this PR, we will keep this and push the new version asap after this is merged. We can figure out a better release structure moving forward.